src/tests/ftest/osa/osa_offline_reintegration.py

```diff
@@ -88,6 +88,9 @@ def run_offline_reintegration_test(self, num_pool, data=False,
                         self.simple_exclude_reintegrate_loop(rank[val])
                     output = self.dmg_command.pool_exclude(self.pool.uuid,
                                                            rank[val])
+                    self.set_container_property(self.pool.uuid,
```

Comment:
(pylint-no-member) Instance of 'OSAOfflineReintegration' has no 'set_container_property' member